service-base/src/test/java/org/eclipse/hono/service/metric/DeviceConnectionDurationTrackerTest.java

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.hono.service.metric;
+
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+/**
+ * Verifies behavior of {@link DeviceConnectionDurationTracker}.
+ *
+ */
+public class DeviceConnectionDurationTrackerTest {
+
+    private Vertx vertx;
+
+    /**
+     * Sets up the fixture.
+     */
+    @BeforeEach
+    public void setup() {
+        vertx = mock(Vertx.class);
+    }
+
+    /**
+     * verifies that the recording of the connection duration is scheduled with the specified interval.
```

Comment:
Please start with a capital letter

Reply:
I have changed and pushed the changes regarding capital letter. Please have a look again.